hw11/Lesson11.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+import inspect
+
+func_name = None
+
+
+def my_decorator_func(func):
+    def deco_func():
+        func()
+        global func_name
+        frame = inspect.currentframe()
+        func_name = inspect.getframeinfo(frame).function
+        now_ = datetime.datetime.now()
+        print(f'The function name is {func_name} - was called at {now_.strftime("%H:%M")}')
+    return deco_func()
```

Comment:
Do not call the wrapper since the code from decorated function will be executed immediately